test/extensions/filters/listener/tls_inspector/tls_inspector_fuzz_test.cc

```diff
@@ -21,11 +21,16 @@ DEFINE_PROTO_FUZZER(
   Stats::IsolatedStoreImpl store;
   ConfigSharedPtr cfg;
 
-  if (input.max_size() == 0) {
-    // If max_size not set, use default constructor
-    cfg = std::make_shared<Config>(*store.rootScope(), input.config());
-  } else {
-    cfg = std::make_shared<Config>(*store.rootScope(), input.config(), input.max_size());
+  try {
+    if (input.max_size() == 0) {
+      // If max_size not set, use default constructor
+      cfg = std::make_shared<Config>(*store.rootScope(), input.config());
+    } else {
+      cfg = std::make_shared<Config>(*store.rootScope(), input.config(), input.max_size());
+    }
+  } catch (const Envoy::EnvoyException& e) {
```

Comment:
I don't fully understand what's going on, but it seems that there's a [PGV validation](https://github.com/envoyproxy/envoy/blob/badd88c2373cd43dbfa0c17d7cae2cb2989ffe6b/test/extensions/filters/listener/tls_inspector/tls_inspector_fuzz_test.proto#L12) in the fuzzer (65536), but the implementation has a lower limit that triggers an exception (the value in the fuzz test is 65533).
I'm not sure what's the history behind the limit that was added to the fuzz proto, but would it make sense to lower the threshold there?